activestorage/active.py

```diff
@@ -1,15 +1,50 @@
 import os
 import numpy as np
+import yaml
+
+import activestorage
 
 #FIXME: Consider using h5py throughout, for more generality
 from netCDF4 import Dataset
+from pathlib import Path
 from zarr.indexing import (
     OrthogonalIndexer,
 )
 from activestorage.storage import reduce_chunk
 from activestorage import netcdf_to_zarr as nz
 
 
+def _read_config_file(storage_type):
+    """Read config user file and store settings in a dictionary."""
+    base_path = Path(activestorage.__file__).parent
+    if storage_type == "S3":
+        config_file = base_path / Path("config-s3-storage.yml")
+    elif storage_type == "Posix":
+        config_file = base_path / Path("config-Posix-storage.yml")
```

Comment:
For "production" it would be better to pull in config from a location outside of the Python installation.